chore: update doc for new providers (#19224)
MystenLabs/sui: docs/content/snippets/openid-providers.mdx

| Slack        | Yes          |   Yes    |  No      | No       |
| Kakao        | Yes          |   Yes    |  No      | No       |
| Microsoft    | Yes          |   Yes    |  No      | No       |
| AWS (Tenant)*| Yes          |   Yes    |  Yes     | Yes      |
| Karrier One  | Yes          |   Yes    |  Yes     | Yes      |
| Credenza3    | Yes          |   Yes    |  Yes     | Yes      |
| RedBull      | Under review |   No     |  No      | No       |
| Amazon       | Under review |   No     |  No      | No       |
| WeChat       | Under review |   No     |  No      | No       |
| Auth0        | Under review |   No     |  No      | No       |
| Okta         | Under review |   No     |  No      | No       |

> * AWS (Tenant) is supported by Sui but need to be enabled per tenant. Contact us for more information.

Reviewer: ```suggestion
* Sui supports AWS (Tenant) but the provider is enabled per tenant. Contact us for more information.
```
